frontend/src/components/FormInput/ForminputElementSubmitButton.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './ForminputElementSubmitButton.css';
+import PropTypes from 'prop-types';
+
+function ForminputElementSubmitButton({ onClickMethod }: { onClickMethod:()=>void }) {
```

Comment:
Nur ne Frage: ist das { onClickMethod:()=>void } der DefaultValue, wenn onClickMethod nicht übergeben wurde? Aber die prop is ja required? Oder was macht das?